docs/portal/dashboard.md

```diff
@@ -1,9 +1,26 @@
-# Understanding the Dashboard
+# Dashboard
 
-The Dashboard appears when you first log in to the {{title}} portal.
+HDT Dashboard give you information about the traffic of your HDT transports.
 
-Its charts provide a snapshot of your account's traffic. Clicking **View Full Report** in the upper right corner of the chart displays the chart on the [Reports page](</docs/portal/reports.md>), where you can define report parameters, and then view the results on the selected chart.
+If you are a reseller with child accounts, you can select the accounts that the dashboards cover (the parent account only, the child account only, or both the parent and child accounts).
 
-![null](</docs/resources/images/dashboard.png>)
+## Understanding the Dashboard
+
+The Dashboard appears when you first log into the {{title}} portal. The charts provide a snapshot of your account's traffic, covering "Traffic Volume", "Traffic Bandwidth" and "Requests".
+
+If you are a paid customer of **Virtual Private Line** usage, you can also switch between “Total” and “Virtual Private Line” dashboard by clicking the selection button at the upper right corner.
```

Comment:
Please consider the following: "If you are a paid customer of **Virtual Private Line** usage, you can also switch between the “Total” and “Virtual Private Line” dashboard by clicking the selection button at the top-right corner."
OR
"Paid customers of **Virtual Private Line** usage can switch between the “Total” and “Virtual Private Line” dashboards by clicking the selection button at the top-right corner."
OR
"Paid customers of **Virtual Private Line** usage can click the selection button at the top-right corner to switch between the “Total” and “Virtual Private Line” dashboards."
OR
"A selection button at the top-right corner allows paid customers of **Virtual Private Line** usage to switch between the “Total” and “Virtual Private Line” dashboards."

Reply:
Will go with the first one.